site/src/components/WorkspaceStatusBadge/WorkspaceStatusBadge.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import { Story } from "@storybook/react"
+import {
+  MockCanceledWorkspace,
+  MockCancelingWorkspace,
+  MockDeletedWorkspace,
+  MockDeletingWorkspace,
+  MockFailedWorkspace,
+  MockQueuedWorkspace,
+  MockStartingWorkspace,
+  MockStoppedWorkspace,
+  MockStoppingWorkspace,
+  MockWorkspace,
+} from "../../testHelpers/renderHelpers"
```

Comment:
```suggestion
} from "testHelpers/renderHelpers"
```

Reply:
I had issues with this and Storybook. Looks like we need to setup Storybook to understand those imports.

Reply:
@BrunoQuaresma can you try merging in main? I pushed a fix up this morning! LMK if you're still having troubles.

Reply:
It is working!